catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformer.java

```diff
@@ -0,0 +1,682 @@
+          "Error handling feature name: {}, with value: {}. No mapping entry found for feature name",
+          featureName,
+          featureValue);
+      return null;
+    }
+
+    AttributeDescriptor attributeDescriptor =
+        metacardType.getAttributeDescriptor(entry.getAttributeName());
+    if (attributeDescriptor == null) {
+      LOGGER.debug(
+          "AttributeDescriptor for attribute name {} not found. The mapping is being ignored.",
+          entry.getAttributeName());
+      return null;
+    }
+
+    Serializable attributeValue = null;
+
+    if (StringUtils.equals(entry.getAttributeName(), Core.RESOURCE_SIZE)) {
+      String bytes = convertToBytes(featureValue, getDataUnit());
+
+      if (StringUtils.isNotBlank(bytes)) {
+        attributeValue = bytes;
+      }
+    } else {
+      attributeValue =
+          getValueForAttributeFormat(
+              attributeDescriptor.getType().getAttributeFormat(), featureValue);
+    }
+
+    return attributeValue;
+  }
+
+  private Serializable getValueForAttributeFormat(
+      AttributeType.AttributeFormat attributeFormat, String value) {
+
+    Serializable serializable = null;
+    switch (attributeFormat) {
+      case BOOLEAN:
+        serializable = Boolean.valueOf(value);
+        break;
+      case DOUBLE:
+        serializable = Double.valueOf(value);
+        break;
+      case FLOAT:
+        serializable = Float.valueOf(value);
+        break;
+      case INTEGER:
+        serializable = Integer.valueOf(value);
+        break;
+      case LONG:
+        serializable = Long.valueOf(value);
+        break;
+      case SHORT:
+        serializable = Short.valueOf(value);
+        break;
+      case XML:
+      case STRING:
+        serializable = value;
+        break;
+      case GEOMETRY:
+        LOGGER.trace("Unescape the geometry: {}", value);
+        value = StringEscapeUtils.unescapeXml(value);
+        LOGGER.debug("Geometry value after it has been xml unescaped: {}", value);
+        String wkt = getWktFromGeometry(value);
+        LOGGER.debug("String wkt value: {}", wkt);
+        serializable = wkt;
+        break;
+      case BINARY:
+        try {
+          serializable = value.getBytes(UTF8_ENCODING);
+        } catch (UnsupportedEncodingException e) {
+          LOGGER.debug("Error encoding the binary value into the metacard.", e);
+        }
+        break;
+      case DATE:
+        try {
+          serializable = DatatypeConverter.parseDate(value).getTime();
+        } catch (IllegalArgumentException e) {
+          LOGGER.debug("Error converting value to a date. value: '{}'", value, e);
+        }
+        break;
+      default:
+        break;
+    }
+    return serializable;
+  }
+
+  private String getWktFromGeometry(String geometry) {
+    String wkt = getWktFromGml3(geometry);
+    if (StringUtils.isNotBlank(wkt)) {
+      return wkt;
+    }
+    LOGGER.debug("Error converting gml to wkt using gml3 configuration. Trying gml2.", geometry);
```

Comment:
missing `{}` for the parameter

Reply:
The gml is being logged elsewhere, so we just removed the `geometry` parameter.